moveit_core/collision_detection_fcl/src/collision_common.cpp

```diff
@@ -495,9 +495,9 @@ bool distanceCallback(fcl::CollisionObjectd* o1, fcl::CollisionObjectd* o2, void
 
   double dist_threshold = cdata->req->distance_threshold;
 
-  const std::pair<const std::string&, const std::string&> pc = cd1->getID() < cd2->getID() ?
```

Comment:
Probably a matter of style, but I believe you could have equivalently removed the references and preserved the ternary.

Reply:
Keeping the (const) references ensure optimization.

Reply:
Yup yup. Was more of a comment before the `std::cref` became the choice thing to do.